accounts/ledger_account.md

```diff
@@ -0,0 +1,23 @@
+## LedgerAccount
+
+```
+class LedgerAccount:
+    // The constructor is not captured by the specs; it's up to the implementing library to define it.
+    // Suggestions:
+    constructor(account_index: int = 0, address_index: int = 0);
+
+    address: Address;
```

Comment:
Maybe simply say that `LedgerAccount` extends `Account` or implement `IAccount` interface, without any other specification?

Reply:
done